src/init.cpp

```diff
@@ -1715,16 +1715,18 @@ bool AppInit2()
         if (fFirstRun) {
             // Create new keyUser and set as default key
             CPubKey newDefaultKey;
-            if (pwalletMain->GetKeyFromPool(newDefaultKey)) {
-                pwalletMain->SetDefaultKey(newDefaultKey);
-                if (!pwalletMain->SetAddressBook(pwalletMain->vchDefaultKey.GetID(), "", "receive"))
-                    strErrors << _("Cannot write default address") << "\n";
+            // Top up the keypool
+            if (!pwalletMain->TopUpKeyPool()) {
+                // Error generating keys
+                InitError(_("Unable to generate initial key") += "\n");
+                LogPrintf("%s %s\n", __func__ , "Unable to generate initial key");
+                return false;
```

Comment:
styling nit: `return error` and no line terminator `\n`.

Reply:
done